device-base/edison/jessie/entry.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+HOSTNAME=$(cat /etc/hostname)
+echo "127.0.1.1 $HOSTNAME" >> /etc/hosts
+hostname $HOSTNAME
+
+if [ "$INITSYSTEM" = "on" ]; then
+	GREEN='\033[0;32m'
+	echo -e "${GREEN}Systemd init system enabled."
+	env > /etc/docker.env
+
+	echo -e "#!/bin/bash\n exec $@" > /etc/resinApp.cmd
```

Comment:
It contains the CMD from the Dockerfile which is then used from systemd here https://github.com/resin-io-library/base-images/blob/master/systemd/launch.service